cypress/integration/individualMoviePage_spec.js

```diff
@@ -30,14 +30,23 @@ describe('Individual Movie Page User Flow', () => {
 
   it('Should return to the home page by clicking the home button', () => {
     cy.get('.individual-movie-container').find('.home-button').click()
-    .url().should('eq', 'http://localhost:3000/')
+      .url().should('eq', 'http://localhost:3000/')
   });
 
-    it('Should route to the trailer page by clicking the Watch Trailers button', () => {
+  it('Should route to the trailer page by clicking the Watch Trailers button', () => {
     cy.get('.individual-movie-container').find('.trailer-link').click()
-    .url().should('eq', 'http://localhost:3000/340102/trailer')
+      .url().should('eq', 'http://localhost:3000/340102/trailer')
+  });
+
+  it('Should go back when pressing back button', () => {
```

Comment:
Fun learning this new piece of Cypress testing that confirms router is working correctly